docs/statistics.md

```diff
@@ -1,269 +1,425 @@
 
-Same as `pktRetransTotal`, but for a specified interval.
+### pktRetrans
 
-## pktRcvRetrans
+Same as `pktRetransTotal`, but for a specified interval. Available for sender.
+
+### pktRcvRetrans
 
 Same as `pktRcvRetransTotal`, but for a specified interval.
 
-## pktSentACK
+TODO: 
+
+- There is no `pktRcvRetransTotal` stats.
+- Which side
+- include in table
+
+### pktSentACK
 
 Same as `pktSentACKTotal`, but for a specified interval.
 
-## pktRecvACK
+TODO: Update accordingly.
+
+### pktRecvACK
 
 Same as `pktRecvACKTotal`, but for a specified interval.
 
-## pktSentNAK
+TODO: Update accordingly.
+
+### pktSentNAK
 
 Same as `pktSentNAKTotal`, but for a specified interval.
 
-## pktRecvNAK
+TODO: Update accordingly.
+
+### pktRecvNAK
 
 Same as `pktRecvNAKTotal`, but for a specified interval.
 
-## pktSndFilterExtra
+TODO: Update accordingly.
 
-Same as `pktSndFilterExtraTotal`, but for a specified interval.
+### pktSndFilterExtra
+
+Same as `pktSndFilterExtraTotal`, but for a specified interval. Sender only.
 
 Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## pktRcvFilterExtra
+### pktRcvFilterExtra
 
-Same as `pktRcvFilterExtraTotal`, but for a specified interval.
+Same as `pktRcvFilterExtraTotal`, but for a specified interval. Receiver only.
 
-Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+Introduced in SRT v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## pktRcvFilterSupply
+### pktRcvFilterSupply
 
-Same as `pktRcvFilterSupplyTotal`, but for a specified interval.
+Same as `pktRcvFilterSupplyTotal`, but for a specified interval. Receiver only.
 
-Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+Introduced in SRT v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## pktRcvFilterLoss
+### pktRcvFilterLoss
 
-Same as `pktRcvFilterLossTotal`, but for a specified interval.
+Same as `pktRcvFilterLossTotal`, but for a specified interval. Receiver only.
 
-Introduced in v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+Introduced in SRT v1.4.0. Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
 
-## mbpsSendRate
+### mbpsSendRate
 
 Sending rate in Mbps. Sender side.
 
-## mbpsRecvRate
+TODO: How it is calculated?
+
+### mbpsRecvRate
 
 Receiving rate in Mbps. Receiver side.
 
-## usSndDuration
+TODO: How it is calculated?
+
+### usSndDuration
 
-Same as `usSndDurationTotal`, but measured on a specified interval.
+Same as `usSndDurationTotal`, but measured on a specified interval. Available for sender.
 
-## pktReorderDistance
+### pktReorderDistance
 
-The distance in sequence numbers between the two original (not retransmitted) packets,
-that were received out of order. Receiver only.
+TODO: How it is calculated? Why it is interval based?
```

Comment:
This is the maximum so far seen distance in sequence numbers between the out-of-order packet (late packet with R=0) and the top receiver sequence. This value is decreased by 1 together with reorder tolerance at the moment when 10 packets in a row were received in order.